src/Admin/AbstractAdmin.php

```diff
@@ -1920,9 +1928,31 @@ public function getChild($code)
         return $this->hasChild($code) ? $this->children[$code] : null;
     }
 
-    public function setParent(AdminInterface $parent)
+    public function setParent(AdminInterface $parent/*, string $parentAssociationMapping*/)
     {
         $this->parent = $parent;
+
+        // NEXT_MAJOR: remove $args and add $field argument.
```

Comment:
```suggestion
        // NEXT_MAJOR: remove $args and add $parentAssociationMapping argument.
```
Keep the same param name. It is only pedantic change.